src/components/Button.js

```diff
@@ -2,17 +2,48 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Button.css';
 
-function Button(props) {
+const Button = props => {
   const { name } = props;
+  const { color } = props;
+  const { wide } = props;
+  const wideStyle = {
+    backgroundColor: color,
+    flexBasis: '50%',
+  };
+  const normalStyle = {
+    backgroundColor: color,
+    flexBasis: '25%',
+  };
   return (
-    <div className="button">
-      <div>{name}</div>
-    </div>
+    wide
+      ? (
+        <div
+          className="button"
+          style={wideStyle}
+        >
+          {name}
+        </div>
+      )
+      : (
+        <div
+          className="button"
+          style={normalStyle}
+        >
+          {name}
+        </div>
+      )
```

Comment:
- [ ] Why not make this an actual button element instead of a div? This will make your application more semantic.

**OPTIONAL**

- [ ] We can simplify this return statement using this code instead:
```suggestion
      <div
        className="button"
        style={wide ? wideStyle : normalStyle}
      >
        {name}
      </div>
```